Makefile

```diff
@@ -17,9 +17,19 @@ build:
 	$(GO_BUILD_RECIPE)
 
 .PHONY: generate
-generate:
+generate: crd
 	hack/update-generated-bindata.sh
 
+# Generate ClusterIngress CRD from vendored API spec.
```

Comment:
`ClusterIngress`→`DNS` here and also for the `verify-crd` target below.

Reply:
Fixed